src/ca/caChannel.cpp

```diff
@@ -1208,6 +1219,12 @@ void CAChannelGet::getDone(struct event_handler_args &args)
         {
             throw  std::runtime_error("CAChannelGet::getDone no copy func implemented");
         }
+        pvCopy->updateMasterSetBitSet(pvStructure,bitSet);
+        if(firstTime) {
+            bitSet->clear();
+            bitSet->set(0);
+            firstTime = false;
```

Comment:
Unrelated to pvCopy, but the 'firstTime' member has a volatile qualifier which looks like it's meant to be used without a mutex.  This looks a bit iffy.  Then it is used here along with changes to ```bitSet``` which certainly isn't reentrant.  This doesn't seem right.  It seems to me that the volatile is unnecessary as it should be covered by any locking needed by ```bitSet``` (which may be none).